config.json

```diff
@@ -642,9 +642,12 @@
       "slug": "diffie-hellman",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Cryptography",
+        "Integers",
+        "Algorithms",
+        "Transforming"
```

Comment:
I think if there are topics we would like to include we should probably propose them to the TOPICS.txt file? I think with nextercism and the way it is managing topics that putting our own in may mess things up? "Standard library" would be a good one, but we would have to be careful not to overuse, since you use some standard library functions a ton. Just where it is unique and not the ubiquitous use of the "fmt" lib.